config/__init__.py

```diff
@@ -22,9 +22,11 @@
 RATELIMIT_STRATEGY = 'moving-window'
 
 DEBUG = False
+SESSION_COOKIE_SECURE = True
 
 if not PRODUCTION:
```

Comment:
Don't you want to extend that to staging too? or change it to `if LOCALHOST:` 
because we introduce a security risk for users.